skopeo/README.md

```diff
@@ -38,3 +38,24 @@ podman run docker://quay.io/skopeo/stable:latest copy docker://quay.io/skopeo/st
 # Inspect the fedora:latest image
 podman run docker://quay.io/skopeo/stable:latest inspect --config docker://registry.fedoraproject.org/fedora:latest  | jq
 ```
+
+## Use against a private registry server is possible but somewhat complex
+
+1. Assuming one isn't already defined, setup a podman secret with the `auth.json` contents.
+   See the `containers-auth.json` man page for the format.
+
+```
+$ REGISTRY_AUTH_FILE=$(mktemp)
+$ podman login example.com/registry
+$ podman secret create registry_name_auth "$REGISTRY_AUTH_FILE"
+$ rm "$REGISTRY_AUTH_FILE"
+```
+
+2. Pass the podman secret into the skopeo container along with the intended command.
+
+```
+$ podman run --secret=registry_name_auth \
+  --env REGISTRY_AUTH_FILE=/run/secrets/registry_name_auth \
+  docker://quay.io/skopeo/stable:latest \
+  inspect docker://example.com/registry/image_name
```

Comment:
`…skopeo… --authfile /run/secrets/registry_name_auth docker://…` would be shorter, more explicit, and has a tiny bit better (but still ~0) chance of being made convenient by tab completion.

Reply:
Ahh yes, I like it.  Thanks for the suggestion.